modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/RichTextLocalized.tsx

```diff
@@ -38,6 +38,7 @@ export function RichTextLocalized({
 	label,
 	onSelectedLocaleChange,
 	onTranslationsChange,
+	readOnly = false,
```

Comment:
@seltong, my friend, why do you pass a variable property (`values.system`) to this component just to overwrite it with a hard-coded value? Why don't we skip passing this prop at all? It is optional in `IProps` and you can pass a `!!readOnly` to `ClassicEditor` to make sure it is a boolean and not undefined. This way, whenever `RichTextLocalized` is used elsewhere, `readOnly` will have a real purpose. Does it make sense?

Reply:
Defining the readOnly property as not required and adding a default value, we don't need to pass the readOnly parameter when `RichTextLocalized` is called because its value is `false` by default.

I'm using the readOnly in the ClassicEditor and FieldBase. In the FieldBase case, I need to pass an inline style as we are doing [here](https://github.com/liferay/liferay-portal/blob/a582cf93c09d3f20be411e4afa58195d14efb145/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js#L168) to disable the "Source" button.

Does it make sense?

Reply:
You are right, @seltong! I will aprove your PR.